src/music.lisp

```diff
@@ -0,0 +1,53 @@
+(in-package #:claylib)
+
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  (defclass rl-music ()
+    ((%c-struct :type claylib/ll:music
+                :initform (autowrap:alloc 'claylib/ll:music)
+                :accessor c-struct)
+     (%pitch :initarg :pitch
+             :type float
+             :reader pitch)
+     (%volume :initarg :volume
+              :type float
+              :reader volume))))
+
+(defcreader astream      rl-music stream      music)
+(defcreader frame-count  rl-music frame-count music)
+(defcreader ctx-type     rl-music ctx-type    music)
+(defcreader ctx-data     rl-music ctx-data    music)
+
+(defcreader-bool looping rl-music looping music)
+(defcwriter-bool looping rl-music looping music)
+
+(defwriter-float pitch rl-music)
+(defmethod (setf pitch) :after (new-value (class rl-music))
+  (claylib/ll:set-music-pitch (c-struct class)
+                              (coerce new-value 'single-float)))
+
+(defwriter-float volume rl-music)
+(defmethod (setf volume) :after (new-value (class rl-music))
+  (claylib/ll:set-music-volume (c-struct class)
+                               (coerce new-value 'single-float)))
+
+(definitializer rl-music
+  :lisp-slots ((%pitch) (%volume))
+  :pt-accessors ((frame-count integer) (looping boolean) (ctx-type integer) #|(ctx-data pointer)|#))
+
+(default-slot-value rl-music %pitch 1.0)
+(default-slot-value rl-music %volume 1.0)
```

Comment:
Just noticed these. `default-slot-value` is "deprecated" in favor of `:default-initargs` on the class definition. There are a couple places we're still using `default-slot-value` only because we can't pass them as default args for other reasons. Is that the case here or did you just overlook?

Reply:
Whoops, just an oversight! Fixed.